onchain/rollups/contracts/inputs/IInputBox.sol

```diff
@@ -7,26 +7,26 @@ pragma solidity ^0.8.8;
 interface IInputBox {
     /// @notice Emitted when an input is added to a DApp's input box.
     /// @param dapp The address of the DApp
-    /// @param inputIndex The index of the input in the input box
+    /// @param index The index of the input in the input box
     /// @param sender The address that sent the input
-    /// @param input The contents of the input
+    /// @param payload The input payload
     /// @dev MUST be triggered on a successful call to `addInput`.
     event InputAdded(
         address indexed dapp,
-        uint256 indexed inputIndex,
+        uint256 indexed index,
         address sender,
```

Comment:
We don't *need* to emit the input sender as argument to the `InputAdded` event anymore, as this information is now encoded in the input itself, and is only really useful to the DApp back-end (while `dapp` and `index` are still useful for the node to process inputs to a DApp and filter/order them by index).

Reply:
We do need if we only emit payload rather than the whole input blob

Reply:
True. Maybe we need to discuss this with the node team.